cups/request.c

```diff
@@ -871,7 +871,7 @@ cupsSendRequest(http_t     *http,	/* I - Connection to server or @code CUPS_HTTP
 	  * Don't try using the Expect: header the next time around...
 	  */
 
-	  expect = (http_status_t)0;
+	  expect = HTTP_STATUS_OK;
```

Comment:
In this case, "0" means "no HTTP status for the Expect: header", but "200" (`HTTP_STATUS_OK`) would mean "send HTTP status 200-OK for the Expect: header". I know the current code only supports 100-continue, but this change could have future side-effects.

This should be changed to `HTTP_STATUS_NONE`.